src/components/SlateEditor/plugins/audio/AudioEmbedForm.tsx

```diff
@@ -92,39 +98,31 @@ const EmbedForm = ({ onCancel, audio }: EmbedFormProps) => {
   const { t } = useTranslation();
   const { isValid, dirty, values } = useFormikContext<FormValues>();
   return (
-    <Form>
-      <FormikField name="type">
-        {({ field }: FieldProps) => (
-          <ObjectSelector
-            onClick={(evt: MouseEvent) => evt.stopPropagation()}
-            onChange={field.onChange}
-            onBlur={field.onChange}
-            key="type"
-            name="type"
-            labelKey="label"
-            idKey="id"
-            value={field.value}
-            options={[
-              {
-                id: "standard",
-                label: t("form.audio.sound"),
-              },
-              {
-                id: "minimal",
-                label: t("form.audio.speech"),
-              },
-            ]}
-          />
+    <StyledForm>
+      <FormField name="type">
+        {({ field, meta }) => (
+          <FormControl isRequired isInvalid={!!meta.error}>
+            <Label textStyle="label-small" margin="none">
+              {t("form.audio.chooseAudioType")}
+            </Label>
+            <Select {...field}>
+              <option value="standard">{t("form.audio.sound")}</option>
+              <option value="minimal">{t("form.audio.speech")}</option>
+            </Select>
+            <FieldErrorMessage>{meta.error}</FieldErrorMessage>
+          </FormControl>
         )}
-      </FormikField>
-      <AudioPlayer src={audio.audioFile.url} title={audio.title.title} speech={values.type === "minimal"} />
+      </FormField>
+      <div>
+        <AudioPlayer src={audio.audioFile.url} title={audio.title.title} speech={values.type === "minimal"} />
+      </div>
```

Comment:
Trengs div rundt denne?

Reply:
Overraskende nok, ja. `AudioPlayer` er to elementer som ikke wrappes av noe, så den påvirkes av gap.